pkg/devfile/adapters/docker/utils/utils.go

```diff
@@ -297,13 +338,18 @@ func StartBootstrapSupervisordInitContainer(client lclient.Client, supervisordVo
 		s = log.Spinnerf("Starting container for %s", image)
 		defer s.End(false)
 	}
-	err = client.StartContainer(&containerConfig, &hostConfig, nil)
+	containerID, err := client.StartContainer(&containerConfig, &hostConfig, nil)
 	if err != nil {
 		return err
 	}
 	if log.IsDebug() {
 		s.End(true)
 	}
 
+	err = client.RemoveContainer(containerID)
```

Comment:
What's the reasoning behind removing the supervisord container?

Reply:
Everytime we init, the container is left behind, thought its exited but you can see a bunch of init containers when you run `docker ps -a`